cadence/contracts/flix_registry.cdc

```diff
@@ -0,0 +1,188 @@
+access(all) 
+contract FLIXRegistry {
+
+    access(all)
+    event ContractInitialized()
+
+    access(all)
+    event RegistryCreated(name: String)
+
+    access(all)
+    event Published(registryOwner: Address, registryName: String, registryUuid: UInt64, alias: String, id: String, cadenceBodyHash: String)
+
+    access(all)
+    event Removed(registryOwner: Address, registryName: String, registryUuid: UInt64, id: String)
+
+    access(all)
+    event Deprecated(registryOwner: Address, registryName: String, registryUuid: UInt64, id: String)
+
+    access(all)
```

Comment:
Sorry if this was covered already, but should these included in the registry owner, name, and UUID as well? It would make it easier on the backend side to resolve conflicts between registries that want to use the same alias for different flix IDs